src/main/resources/application.yml

```diff
@@ -120,28 +136,28 @@ initializr:
           mappings:
           - versionRange: "[1.0.0.RELEASE,2.0.0.RELEASE)"
             version: 1.3.1.RELEASE
-          - versionRange: "[2.0.0.RELEASE,3.0.0.RELEASE)"
-            version: 2.0.0.RELEASE
-            repositories: spring-snapshots,spring-milestones
+          - versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"
+            version: 2.0.2.RELEASE
       field-value-counter-bom:
           groupId: org.springframework.cloud.stream.app
           artifactId: field-value-counter-app-dependencies
           additionalBoms: [app-starters-core-bom]
           mappings:
           - versionRange: "[1.0.0.RELEASE,2.0.0.RELEASE)"
             version: 1.3.1.RELEASE
-          - versionRange: "[2.0.0.RELEASE,3.0.0.RELEASE)"
-            version: 2.0.0.RELEASE
-            repositories: spring-snapshots,spring-milestones
+          - versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"
```

Comment:
We dropped field-value-counter in the 2.1.0.RELEASE.

Reply:
`- versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"` means excluding `2.1.0.RELEASE`